### DISCUSSION TITLE
how to show artist in file name

### DISCUSSION THREAD

**Discussion by dreedlord91** [shaked6540/YoutubePlaylistDownloader#191] [Q&A]
Hi. What do i need to type in the file file name pattern to see the artist in the file name ?
I already tried $title-$artist but it will not show.
i tried all of them in the help section and it wont show me the artist.. I am trying to get this playlist https://music.youtube.com/watch?v=ujG5c5RmQJg&list=PLKVGj6Z3oDC_v6sa9s4YjXG-5OIKM_KSQ

**Comment by shaked6540** [comment]
The artist doesn't show in any of these videos? I'll check it over the weekend

**Reply by dreedlord91** [reply]
I mean it shows under the name of the song but i dont know why it wont out the name in the file name.
Thank you for your hard work <3

**Reply by shaked6540** [reply]
Hi, just letting you know that I've identified the problem. It only happens with youtube music, a fix should be out next weekend if I have time. Sorry for the delays, I'm busy :\

**Reply by dreedlord91** [reply]
It`s fine don`t worry i can wait. Thanks for the hard work <3

**Reply by dreedlord91** [reply]
Hi have you managed to make the change ?

**Reply by shaked6540** [reply]
Not yet sadly

**Reply by dreedlord91** [reply]
Hi, any update on the change ? :D

And if possible can you make it so that when the song that I download is already downloaded in the folder to skip it?

**Reply by shaked6540** [reply]
Hey, sorry but there won't be an update any time soon because of the war in Israel, I barely have time to respond to new issues, let alone code. 
Its already possible to skip files that are already downloaded. check the "skip files that already exist" in the options: 
![image](https://github.com/shaked6540/YoutubePlaylistDownloader/assets/13851815/a2988c35-693c-498f-b9aa-9bcb8aaafaaa)
